src/poetry/repositories/legacy_repository.py

```diff
@@ -62,7 +62,9 @@ class Page:
         ".tar",
     ]
 
-    def __init__(self, url: str, content: str, headers: Dict[str, Any]) -> None:
+    def __init__(
+        self, url: str, content: bytes, headers: requests.structures.CaseInsensitiveDict
```

Comment:
I would prefer that we stuck to `Dict` or `Mappting`. We do want to drop requests at some point.

Reply:
switched to mapping, mypy was complaining about the dict.